code/modules/clothing/glasses/night.dm

```diff
@@ -70,6 +70,14 @@
 	name = "\improper M42 spotter sight"
 	desc = "A companion headset and night vision goggles system for USCM spotters. Allows highlighted imaging of surroundings. Click it to toggle."
 
+/obj/item/clothing/glasses/night/m42_night_goggles/spotter/can_use_active_effect(mob/living/carbon/human/user)
+	if(!(GLOB.character_traits[/datum/character_trait/skills/spotter] in user.traits))
```

Comment:
This shouldn't be exclusive to spotter. No reason why the spec can't also use it.